src/endpoint/endpoint.js

```diff
@@ -349,6 +355,21 @@ function fork_count_handler(req, res) {
     res.end(JSON.stringify(reply));
 }
 
+/**
+ * version_handler return the version of noobaa package
+ * @param {*} req 
+ * @param {*} res 
+ */
+function version_handler(req, res) {
+    const reply = {
+        version: pkg.version,
+    };
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'application/json');
+    res.setHeader('Content-Length', Buffer.byteLength(JSON.stringify(reply)));
+    res.end(JSON.stringify(reply));
```

Comment:
When returning a json it would be better to prepare the buffer once and not call stringify twice
```suggestion
    const buffer = Buffer.from(JSON.stringify(reply));
    res.statusCode = 200;
    res.setHeader('Content-Type', 'application/json');
    res.setHeader('Content-Length', buffer.length);
    res.end(buffer);
```

Reply:
sure, I also fixed the places where I copied it from